src/tests/tribol_mortar_lm_patch_test.cpp

```diff
@@ -281,7 +277,7 @@ void MortarLMPatchTest::computeContactSolution( int nMortarElemsX, int nMortarEl
          int offset = this->m_mesh.dim * (this->m_mesh.numMortarNodes + this->m_mesh.numNonmortarNodes);
          int nonmortarOffset = this->m_mesh.numMortarNodes;
          int id = this->m_mesh.faceConn2[i];
-         nonmortarMesh.m_nodalFields.m_node_pressure[id] = sol_data[ offset + id - nonmortarOffset ];
+         this->m_mesh.pressures[id] = sol_data[ offset + id - nonmortarOffset ];
```

Comment:
Was this just staying consistent between the TestMesh class data vs. the Tribol MeshData class data? Here we want to use the TestMesh?

Reply:
I made `m_node_pressure` `const` in MeshData (see L35 of MeshData.hpp), since Tribol shouldn't be updating `m_node_pressure`, the host code should.  `this->m_mesh.pressures` is the (non-const) "host code" storage of `m_node_pressure` (registered in L858 of TestUtils.cpp).